docs/Segments.md

```diff
@@ -200,8 +196,29 @@ If you migrate symbols from assembly to C, please prefix the `rodata` with a `.`
   start: 0xABC
 ```
 
-`splat` will not generate `.rodata.s` files for these sections, as the symbols should be declared in the C file specified by `filepath`.
+**NOTE:** `splat` will not generate any `.rodata.s` files for these `.` (dot) sections.
+
+## `bss`
+
+**Description:**
 
+`bss` is where variables are placed that have been declared but are not given an initial value. These sections are usually discarded from the final binary (although PS1 binaries seem to include them!).
```

Comment:
Should mention the parent segment needs to specify the `bss_size`, otherwise splat can't disassemble bss at all